cmd/init/init.go

```diff
@@ -128,7 +128,7 @@ func Run(namespace, k8sContext, devPath, language, workDir string, overwrite boo
 		} else {
 			dev.Container = container
 			if container == "" {
-				container = r.PodTemplateSpec.Spec.Containers[0].Name
+				container = r.GetPodTemplate().Spec.Containers[0].Name
```

Comment:
What's the difference between two options? and why the new one fixes the e2e tests?